furusiya/model/systems/ai_system.py

```diff
@@ -21,3 +16,9 @@ def get_ai(cls, owner):
     @classmethod
     def has_ai(cls, owner):
         return cls.get_ai(owner) is not None
+
+    @classmethod
```

Comment:
Can this be a `staticmethod` instead? If `cls` is always the same, we can just reference it directly (eg. `GameObject.get_ai`).

Reply:
Probably. A `classmethod` works just fine as well. I don't see a reason to change it.